app/src/main/java/ch/epfl/sdp/blindly/user/User.kt

```diff
@@ -200,14 +199,14 @@ class User private constructor(
         fun DocumentSnapshot.toUser(): User? {
             try {
                 val username = getString("username")!!
-                val location = getString("location")!!
+                val location = get("location") as List<Double>
```

Comment:
I think it is a bit risky to do a unsafe cast here, imagine a user put an invalid list of Strings in the db, could they make every app that encounters them while swiping crash? Maybe a safe cast is the way to go: [https://kotlinlang.org/docs/typecasts.html#safe-nullable-cast-operator](https://kotlinlang.org/docs/typecasts.html#safe-nullable-cast-operator)

Reply:
I don't know, to me it makes more sense to check the inputs, so that no invalid inputs are put on the database in the first place

Reply:
Of couse but a malicious user could still send queries from outside the app to put invalid data in our db, especially as we aren't doing any kind of server-side validation of the data.